src/utils/getExportTypes.ts

```diff
@@ -596,6 +642,162 @@ const getExportTypes = ({
         ];
       },
     },
+    {
+      name: "PDF",
```

Comment:
it looks like this method is just doing the markdown export and then converting to pdf - is that right? could we reuse?

Reply:
Much of it, yes, but not all.  PDF doesn't use the YAML functionality.

I actually started to move these functions out (as I did with `getExportSettings()`):
-  `treeNodeToMarkdown()`
- `getMarkdownContent()`
- `getDiscourseResultsContent()`
- `getReferenceResultsContent()`

but with the amount of rewrite to the markdown callback, it started to feel like it deserved it's own PR.

Should I include that with this PR or would adding a `// TODO` suffice?

Reply:
discussed offline, good to defer